src/tests/Microsoft.PowerFx.Connectors.Tests/BasicRestTests.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.PowerFx.Core.Tests;
+using Xunit;
+
+namespace Microsoft.PowerFx.Tests
+{
+    // Simulate calling basic REST services, such as ASP.Net + Swashbuckle. 
+    public class BasicRestTests : PowerFxTest
+    {
+        // Must set the BaseAddress on an httpClient, even if we don't actually use it. 
+        // All the Send() methods will enforce this. 
+        private static readonly Uri _fakeBaseAddress = new Uri("http://localhost:5000");
+                
+        [Fact]
+        public async Task BasicHttpCall()
+        {
+            var testConnector = new LoggingTestServer(@"Swagger\TestOpenAPI.json");
+
+            var httpClient = new HttpClient(testConnector)
+            {
+                BaseAddress = _fakeBaseAddress
+            };
+
+            var config = new PowerFxConfig();
+            var apiDoc = testConnector._apiDocument;
+            
+            config.AddService("Test", apiDoc, httpClient);
+
+            var engine = new RecalcEngine(config);
+
+            testConnector.SetResponse("55");
+            var r1 = await engine.EvalAsync("Test.GetKey(\"Key1\")", CancellationToken.None);            
+            Assert.Equal(55.0, r1.ToObject());
```

Comment:
Power Fx is translating a functional expression like:
`Test.GetKey("Key1") `

into a REST call using the swagger.